tests/unit/core/server/testServer.js

```diff
@@ -83,9 +100,66 @@ describe('core/server/base', () => {
       );
 
       const response = await testClient({ stubRoutes })
-        .get('/en-US/firefox/simulation-of-a-non-existent-page').end();
+        .get('/en-US/firefox/simulation-of-a-non-existent-page')
+      ;
```

Comment:
for consistency with the rest of the code base, the semi-colon should always go at the end of the line.

We'll add a rule 😇 https://github.com/mozilla/eslint-config-amo/issues/24